.gitignore

```diff
@@ -1,16 +1 @@
-# Node rules:
-## Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
-.grunt
-
-## Dependency directory
-## Commenting this out is preferred by some people, see
-## https://docs.npmjs.com/misc/faq#should-i-check-my-node_modules-folder-into-git
-node_modules
-
-# Book build output
-_book
-
-# eBook build output
-*.epub
-*.mobi
-*.pdf
+target/
```

Comment:
The gitignore file was fine i think, if you need to add 'target/' to the file you can do so. However, deleting every gitignore rule is not OK. Please restore the gitignore file and add 'target/' to it.